app/assets/stylesheets/dashboard/index.scss

```diff
@@ -0,0 +1,36 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 0.5fr 1fr 1.5fr;
+  gap: 10px 0px;
+  grid-template-areas:
+    "profile log-form log-form log-form"
+    "med-list recent-logs recent-logs recent-logs";
+}
+
+.sidebar {
+  height: 100%;
+  position: fixed;
+}
```

Comment:
I was trying to emulate [this](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_sidenav_fixed) type of side bar to contain the profile and meds list details on the dashboard, but I'm not sure that's actually the right move. Left it for now since it doesn't make a difference at this point.